python/pyarrow/hdfs.py

```diff
@@ -27,8 +27,9 @@
 
 class HadoopFileSystem(lib.HadoopFileSystem, FileSystem):
     """
-    FileSystem interface for HDFS cluster. See pyarrow.hdfs.connect for full
-    connection details
+    FileSystem interface for HDFS cluster.
+
+    See pyarrow.hdfs.connect for full connection details
```

Comment:
This type of change seems overly pedantic, can we turn this warning off?

Reply:
Although here it's maybe less useful, for longer docstrings I personally find the general rule of "first line should be a short summary" a good rule

The first sentence is also used for autosummary tables (although in this case the first sentence *is* short)

Reply:
I also like the short summary, but of course we can turn it off by default.